src/main.cpp

```diff
@@ -1058,7 +1056,7 @@ bool CheckTransaction(const CTransaction& tx, CValidationState &state,
         }
     }
 
-    if (!scMgr.checkTransaction(tx, state) )
+    if (!Sidechain::ScMgr::checkTxSemanticValidity(tx, state) )
```

Comment:
This is static since only semantic validity is controlled, in other words no use is made of side chain state (which btw is managed by the singleton instance)

Reply:
ok. I got it, thanks